### ISSUE TITLE
ML: VGG Model #1 (24 Layers): Configuration#8: Training model and store the results in the directory for 30 and 50 epochs

### ISSUE THREAD

**Issue by hmehryar** [hmehryar/Hm.JetscapeMl#272]
Model Architect: 

- 4Conv(Conv+Conv+MaxPool+Dropout)
- 4Dense(Dense+Dropout)

1. Put the training on the Grid by using jetscape-ml-tensorflow-cnn.sh
2. save the model and the evaluation data in the power point [slides](https://docs.google.com/presentation/d/1H7D-TKbmZjwNFNcf8Ecuz3HLl1veCZVL/edit?usp=sharing&ouid=108370064464811889896&rtpof=true&sd=true)
3. update the diagrams data on the power point slides

**Comment by hmehryar** [comment]
30 epochs

Epoch 00030: val_accuracy did not improve from 0.93061

840000/840000 [==============================] - 312s 371us/sample - loss: 0.8870 - accuracy: 0.9268 - val_loss: 0.8839 - val_accuracy: 0.9147

**Comment by hmehryar** [comment]
50 epochs

Epoch 00034: val_accuracy did not improve from 0.93296

840000/840000 [==============================] - 313s 372us/sample - loss: 0.7865 - accuracy: 0.9267 - val_loss: 0.7826 - val_accuracy: 0.9169
Epoch 00034: early stopping

**Comment by hmehryar** [comment]
port into Excel for overleaf
